### ISSUE TITLE
TrustGraph CLI (in Rust)

### ISSUE THREAD

**Issue by harlantwood** [citizencode/trust-exchange#31]
# TrustGraph CLI Design notes

- other output formats
  - other JSON-LD formats

- "extra" fields for all the things?

- "flavors" of TrustAtoms, including Journalistic flavor: https://github.com/blueskyCommunity/OpenTrustClaims/blob/gvelez17-harlan-trustgraph-compatible/open_trust_claim.yaml

**Comment by gvelez17** [comment]
Yes!  committed it here - https://github.com/blueskyCommunity/OpenTrustClaims/blob/main/open_trust_atom.yaml

**Comment by gvelez17** [comment]
would love to contribute a way to support these fields
